internal/profiles/models/models.go

```diff
@@ -15,6 +15,75 @@
 // Package models contains domain models for profiles
 package models
 
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/google/uuid"
+
+	"github.com/stacklok/minder/internal/db"
+	minderv1 "github.com/stacklok/minder/pkg/api/protobuf/go/minder/v1"
+)
+
+// ProfileAggregate represents a profile along with rule instances
+type ProfileAggregate struct {
+	ID           uuid.UUID
+	Name         string
+	ActionConfig ActionConfiguration
+	Rules        []RuleInstance
```

Comment:
This type has `Aggregate` in the name because it contains a list of rule instances as well as just representing the profile itself.